pynumdiff/tests/test_diff_methods.py

```diff
@@ -174,18 +175,19 @@ def test_diff_method(diff_method_and_params, test_func_and_deriv, request): # re
         else diff_method(x_noisy, dt, params, options)
     
     # check x_hat and x_hat_noisy are close to x and that dxdt_hat and dxdt_hat_noisy are close to dxdt
-    #print("]\n[", end="")
+    if request.config.getoption("--bounds"): print("]\n[", end="")
     for j,(a,b) in enumerate([(x,x_hat), (dxdt,dxdt_hat), (x,x_hat_noisy), (dxdt,dxdt_hat_noisy)]):
-        # l2_error = np.linalg.norm(a - b)
```

Comment:
I got tired of commenting and uncommenting these lines and then sometimes forgetting and committing the file with lines commented out. A command line arg is easier.